### DISCUSSION TITLE
Fire and evac

### DISCUSSION THREAD

**Discussion by dleone81** [dleone81/apcelettro#44] [General]
Mi serve avere il contenuto della pagina

**Comment by fabiocrazycurls** [comment]
Questo è da creare da zero in quanto la mandante è in via di definizione

**Comment by dleone81** [comment]
Appena creato il contenuto caricatelo qui

**Comment by dleone81** [comment]
Ciao @Cattalek93 , @fabiocrazycurls  ho rimosso il contenuto dal menu, quando sarà pronto il contenuto lo re-inserirò
